db/system_keyspace.cc

```diff
@@ -1811,6 +1811,95 @@ class nodetool_status_table : public memtable_filling_virtual_table {
     }
 };
 
+class describe_ring_table : public streaming_virtual_table {
```

Comment:
Tables names should be nouns, not verbs. So ring_description or even just "topology".

Reply:
not part of this PR, explained below